test/val/val_decoration_test.cpp

```diff
@@ -2303,18 +2341,18 @@ TEST_F(ValidateDecorations, MultiplePushConstantsSingleEntryPointGood) {
                 OpMemoryModel Logical GLSL450
                 OpEntryPoint Fragment %1 "main"
                 OpExecutionMode %1 OriginUpperLeft
-    
```

Comment:
Do I need to be worried about these auto indentations which are done by my editor(Sublime)?

Reply:
Not as long as the tests keep passing and the format checker is ok.